xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.469.xml

```diff
@@ -107,7 +113,12 @@
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
+            <lb n="1"/> <gap reason="lost" extent="unknown" unit="character"/> <persName><name><expan><abbr>Crocal<ex>a</ex>e</abbr></expan></name>
```

Comment:
Is this an abbreviation or just a Greek dative?

Reply:
Could be! I will remove the abbreviation.